### ISSUE TITLE
[Feature Request] "Migration Guide" from TravisCI to LambCI

### ISSUE THREAD

**Issue by joemccann** [lambci/lambci#21]
Completely self-serving for github.com/joemccann/dillinger

Would prerfer to build against LambCI instead of TravisCI

**Comment by mhart** [comment]
Yeah, good idea – I mean, basically, everything's done in one single command in LambCI – there's no separate phases or anything like that – I find that all a bit confusing to be honest w/ Travis, CircleCI, etc – it just seems easier to me to put that stuff in scripts in your package.json

But yeah, would be good to have it documented I guess.

**Comment by joemccann** [comment]
Right the docs would be nice...I'm gonna give it a go

On Friday, July 8, 2016, Michael Hart <notifications@github.com> wrote:

> Yeah, good idea – I mean, basically, everything's done in one single
> command in LambCI – there's no separate phases or anything like that – I
> find that all a bit confusing to be honest w/ Travis, CircleCI, etc – it
> just seems easier to me to put that stuff in scripts in your package.json
>
> But yeah, would be good to have it documented I guess.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lambci/lambci/issues/21#issuecomment-231513167>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAC8alqOBnzHVPQTwqi0bISGLTa-Z1sUks5qTxqmgaJpZM4JIhEw>
> .
>


-- 
Sent from Joe In Real Life
